src/sentry/apidocs/examples/project_examples.py

```diff
@@ -110,6 +110,324 @@
     "status": "active",
 }
 
+detailed_project = {
+    "id": "4505278496",
+    "slug": "pump-station",
+    "name": "Pump Station",
+    "platform": "python",
+    "dateCreated": "2021-01-14T22:08:52.711809Z",
+    "isBookmarked": False,
+    "isMember": True,
+    "features": [
+        "alert-filters",
+        "custom-inbound-filters",
+        "data-forwarding",
+        "discard-groups",
+        "minidump",
+        "race-free-group-creation",
+        "rate-limits",
+        "servicehooks",
+        "similarity-indexing",
+        "similarity-view",
+        "span-metrics-extraction",
+        "releases",
+    ],
+    "firstEvent": "2021-01-14T22:08:52.711809Z",
+    "firstTransactionEvent": True,
+    "access": [
+        "project:releases",
+        "event:read",
+        "project:read",
+        "member:read",
+        "team:read",
+        "event:write",
+        "org:read",
+        "alerts:read",
+    ],
+    "hasAccess": True,
+    "hasMinifiedStackTrace": False,
+    "hasMonitors": True,
+    "hasProfiles": True,
+    "hasReplays": False,
+    "hasSessions": True,
+    "isInternal": True,
+    "isPublic": False,
+    "avatar": {"avatarType": "letter_avatar", "avatarUuid": None},
+    "color": "#3fbf67",
+    "status": "active",
+    "team": {"id": "2", "name": "Powerful Abolitionist", "slug": "powerful-abolitionist"},
+    "teams": [{"id": "2", "name": "Powerful Abolitionist", "slug": "powerful-abolitionist"}],
```

Comment:
Not related to your code changes, but what is the difference between team and teams?

Reply:
Teams can have multiple listed, while Team is always a single team. I'm not too sure what the difference is, but it's what's currently returned by the API and was previously defined in our schema like this

Reply:
backward compatibility maybe? teams should include team anyways so I suggest removing team (not here, another PR). @leedongwei do you have any thoughts on if this change would be safe?